worker.py

```diff
@@ -721,6 +722,7 @@ def accurate_speed(self, point):
         result = responses.get('FORT_SEARCH', {}).get('result', 0)
         if result == 1:
             self.logger.info('Spun {}.'.format(name))
+            self.next_spin = time() + config.SPIN_COOLDOWN
```

Comment:
I would probably prefer for this to be set regardless of the result. If for some reason a worker is repeatedly failing to spin then I wouldn't want that wasting more requests.